### DISCUSSION TITLE
升级到2.4.3后无法自定义评论分页

### DISCUSSION THREAD

**Discussion by leirock** [ArtalkJS/Artalk#305] [Q&A] [answered]
dokcer 部署在自己的服务器，用 [hexo-next-artalk](https://github.com/leirock/hexo-next-artalk) 引入前端，并设置了用后端配置前端。

后端的 `artalk-go.yml` 中设置如下，升级到2.4.3后，目前依然显示默认的 20 条评论，而非 5 条评论。之前版本是正常的。

```yml
frontend:
  pagination:
    pageSize: 5
```

参考链接：https://pinlyu.com/guestbook/
博客源代码中的 [artalk 配置](https://github.com/leirock/blog/blob/23a1ce027dca6a462f8feb1383056e9128791d3c/_config.yml#L143-L154)

**Comment by qwqcode** [comment, marked as answer]
这是一个 BUG，目前可以在前端配置 `new Artalk({ pagination: { pageSize: 5 } })` 临时解决

**Reply by leirock** [reply]
后期会解决吗，会的话那我就暂时不用这个临时方案处理，等版本升级好了。

**Reply by qwqcode** [reply]
已修复，晚些时候发布更新

https://github.com/ArtalkJS/ArtalkGo/commit/3d1cbae1630af08cf3732405d2fef6a2692e5b56

**Reply by qwqcode** [reply]
ok，搞定，补充升级已发布

https://artalk.js.org/guide/backend/update.html

更新后执行 `./artalk-go version` 应为 `v2.3.0/3d1cbae`

**Reply by qwqcode** [reply]
新版本这些配置都能直接在侧边栏修改 更方便了 😁
